_posts/de/newsletters/2022-10-26-newsletter.md

```diff
@@ -0,0 +1,357 @@
+        [Überblick][zhao overview] über die Situation.
+
+
+    - *Keine Entfernung bedeutet, dass das Problem auftreten könnte:* In der
+      IRC-Diskussion [wiederholte][towns uncoordinated] Anthony Towns seine
+      Aussagen von letzter Woche: "Wenn wir die Option `mempoolfullrbf` nicht
+      aus 24.0 entfernen, wird eine unkoordinierte Einführung stattfinden."
+
+        Greg Sanders war [skeptisch][sanders doubt]: "Die Frage ist: Werden 5%+
+        eine Variable setzen? Ich vermute nicht." Towns [entgegnete][towns uasf]:
+        "[UASF][topic soft fork activation] `uacomment` hat gezeigt, dass es
+        einfach ist, ~11% in nur ein paar Wochen dazu zu bewegen, eine Variable
+        zu setzen".
+
+    - *Sollte eine Option sein:* Martin Zumsande [sagte][zumsande option] in der
+      IRC-Diskussion: "Ich denke, wenn eine bedeutende Anzahl von
+      Knoten-Betreiber und Miner eine bestimmte Richtlinie wünscht, sollte es
+      nicht den Entwicklern obliegen, ihnen zu sagen: 'Die könnt ihr jetzt
+      nicht haben'. Die Entwickler können und sollten eine Empfehlung
+      aussprechen (indem sie die Standardeinstellung setzen), aber es sollte nie
+      ein Problem sein, informierten Nutzern Optionen zu bieten."
+
+
+    Zum Zeitpunkt der Erstellung dieses Artikels war noch keine eindeutige
+    Lösung erreicht worden. Die Option `mempoolfullrbf` ist immer noch in den
+    Release Candidates für die kommende Version von Bitcoin Core 24.0 enthalten
+    und es ist Optechs Empfehlung, dass jeder Dienst, der von
+    Nullbestätigungs-Transaktionen abhängt, die Risiken sorgfältig abwägt,
+    vielleicht beginnend mit dem Lesen der E-Mails, die im [Newsletter von letzter
+    Woche][news222 rbf] verlinkt sind.
+
+- **CoreDev.tech Transkripte:** vor der Atlanta Bitcoin Conference (TabConf)
+  nahmen etwa 40 Entwickler an einer CoreDev.tech Veranstaltung teil.
+  [Transkripte][coredev xs] für etwa die Hälfte der Veranstaltungstreffen wurden
+  von Bryan Bishop zur Verfügung gestellt. Zu den bemerkenswerten Diskussionen
+  gehörten:
+
+    - [Transportverschlüsselung][p2p encryption]: ein Gespräch über die jüngste
+      Aktualisierung des Vorschlags für ein verschlüsseltes
+      [Transportprotokoll der Version 2][topic v2 p2p transport] (siehe
+      [Newsletter #222][news222 bip324]). Dieses Protokoll würde es Lauschern im
+      Netz erschweren, herauszufinden, von welcher IP-Adresse eine Transaktion
+      ausgeht, und die Fähigkeit verbessern, Man-in-the-Middle-Angriffe zwischen
+      ehrlichen Knoten zu erkennen und abzuwehren.
+
+        Die Diskussion behandelt mehrere Überlegungen zum Protokollentwurf und
+        ist eine empfehlenswerte Lektüre für alle, die sich fragen, warum die
+        Protokollautoren bestimmte Entscheidungen getroffen haben. Außerdem wird
+        der Zusammenhang mit dem früheren Protokoll zur Authentifizierung durch
+        [Gegenzeichnung][topic countersign] untersucht.
```

Comment:
“Countersign” is a proper noun (Eigenname), so maybe it should not be translated.